packages/compartment-mapper/src/import-archive.js

```diff
@@ -30,13 +31,38 @@ const q = JSON.stringify;
 
 const textDecoder = new TextDecoder();
 
+const freeze = Object.freeze;
```

Comment:
Prevailing style is `const { freeze } = Object` unless you need a TypeScript JSDoc (which you might need to get tsc off your back about ReadOnly!)

Reply:
I'll solve this once the mechanics of firing the error are done. Copy-pasted this from where the typing was necessary.